package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@lanaco/lnc-react-ui",
   "type": "module",
-  "version": "3.7.0",
```

Comment:
Following semantic versioning, new features that does not break backward compatibility should update minor version (digit in the middle). The third digit is reserved for bugfixes only, while the first digit is for major versions that contain breaking changes...